### ISSUE TITLE
Gradle, Maven?

### ISSUE THREAD

**Issue by tukorama** [Whichhunter404/2048#1]
Could you add Gradle or Maven to the project to contribute easily?

**Comment by SchmidtPeti** [comment]
Yes, I tend to agree with you about this issue, but still you are also a contributor so If it bothers you that much than you are free to fix it any time.
